internal/pkg/client/storageos/common/constant.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import "time"
+
+const (
+	// APIUsernameKey is the name of the k8s secret key that stores StorageOS
+	// username.
+	APIUsernameKey = "apiUsername"
+	// APIPasswordKey is the name of the k8s secret key that stores StorageOS
+	// password.
+	APIPasswordKey = "apiPassword"
+	// UserAgent is the user-agent name of the StorageOS client.
+	UserAgent = "cluster-operator"
```

Comment:
Would it be possible to include the operator version in the user agent? It would be really helpful!

Reply:
Yes, that would be nice. Will have to add it to release automation.